pages/common/composer.md

```diff
@@ -0,0 +1,23 @@
+# composer
+
+> A per-project package manager for PHP.
```

Comment:
Do we need to say "per-" ? I think "project manager for PHP applications" should be good enough ?